ChangeLog

```diff
@@ -1,5 +1,11 @@
 mbed TLS ChangeLog (Sorted per branch, date)
 
+= mbed TLS x.x.xx branch released xxxx-xx-xx
```

Comment:
We know already that the version will start with 1.3 and therefore we usually write `1.3.x` instead of `x.x.x` here.